scripts/ci/manage-operator.sh

```diff
@@ -236,13 +237,32 @@ spec:
 ${INSTALL_OBJECTS}
 EOF
 
+OWNER_AND_BRANCH_LOCATION=${OWNER_AND_BRANCH_LOCATION:-codeready-toolchain/toolchain-cicd/master}
+
+start_collecting_logs
 wait_until_is_installed
 }
 
+
+start_collecting_logs() {
+    COLLECT_LOGS=scripts/ci/collect-logs.sh
+    PARAMS="-n ${NAMESPACE}"
```

Comment:
```suggestion
    PARAMS="-n ${NAMESPACE}"
```
Doesn't look like this is being used anywhere?

Reply:
right, but your suggestion doesn't contain any changes :-)

Reply:
removed in 806173c